src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,88 @@
+package duke;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.*;
```

Comment:
You might want to list out the specific imports instead of using a wildcard import